src/test/java/com/dougnoel/sentinel/elements/ElementTests.java

```diff
@@ -216,7 +216,7 @@ public void checkElementBackgroundColorDefaultsWhiteIfAllTransparency() {
     @Test(expected = NoSuchElementException.class)
     public void tableColumnDoesNotExist() {
         BaseSteps.navigateToPage("TablePage");
-        TableVerificationSteps.verifyCellInSpecifiedRow("1", "Not a real column", "example table", "contains", "Bob");
+        TableVerificationSteps.verifyCellInSpecifiedRow("1", "Not a real column", "example table", "","contains", "Bob");
```

Comment:
Is this intentional?

Reply:
yup. change in method signature. the empty string is for the "negate" argument of `verifyCellInSpecifiedRow()`. The old code here is actually wrong, since it should be an empty string instead of "contains". Before this branch, there was no option for "contains" vs "has/had".